addons/bus/models/res_partner.py

```diff
@@ -13,6 +13,10 @@ def _compute_im_status(self):
         status_by_partner = {}
         for presence in self.env["bus.presence"].search([("user_id", "in", self.user_ids.ids)]):
             partner = presence.user_id.partner_id
+            forced_im_status = presence.user_id.forced_im_status
```

Comment:
You cannot use a field that is declared in the mail module in the bus module. IM status should be in mail altogether, but it's not the case. I guess you have to override the compute method in the mail module.